WinUIGallery/Pages/AllControlsPage.xaml.cs

```diff
@@ -20,10 +20,10 @@ public AllControlsPage()
 
     protected override void OnNavigatedTo(NavigationEventArgs e)
     {
-        if (e.Parameter is NavigationRootPageArgs args && args.NavigationRootPage.NavigationView.MenuItems.OfType<NavigationViewItem>().FirstOrDefault(item => item.Name == "AllControlsItem") is NavigationViewItem item)
-        {
-            item.IsSelected = true;
-        }
+        //if (e.Parameter is NavigationRootPageArgs args && args.NavigationRootPage.NavigationView.MenuItems.OfType<NavigationViewItem>().FirstOrDefault(item => item.Name == "AllControlsItem") is NavigationViewItem item)
```

Comment:
Why is this commented out?

Reply:
Oversight, I guess! Would you mind taking a look at this to get this PR in?